tests/kafkatest/services/console_consumer.py

```diff
@@ -99,7 +99,7 @@ class ConsoleConsumer(JmxMixin, BackgroundThreadService):
             "collect_default": True}
     }
 
-    def __init__(self, context, num_nodes, kafka, topic, new_consumer=False, message_validator=None,
+    def __init__(self, context, num_nodes, kafka, topic, new_consumer=False, message_validator=is_int,
```

Comment:
This seems like a step backwards, and we need to be very careful changing defaults like this. I think this actually *used* to be `is_int` at some point and was converted to `None` because `is_int` is highly specific to a particular source of data -- `VerifiableProducer`. Changing this default to `is_int` probably breaks other tests. For example, the Copycat tests uses this, but they're just sending strings through, not integers.